.github/workflows/python-package.yml

```diff
@@ -25,19 +29,21 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
```

Comment:
I changed it to use `uv` instead to install the package, which is much faster than pip so it will reduce the CI time a bit. If you mind, I can revert.

Reply:
Yes, please revert.  I want to minimize the number of dependencies to those that are absolutely essential.